src/lib/pickles/wrap_verifier.ml

```diff
@@ -432,33 +444,45 @@ struct
     end
 
     let combine batch ~xi without_bound with_bound =
+      let reduce_point p =
+        let point = ref (Point.underlying p.(Array.length p - 1)) in
+        for i = Array.length p - 2 downto 0 do
+          point := Point.add p.(i) (Scalar_challenge.endo !point xi)
+        done ;
+        !point
+      in
       let { Curve_opt.non_zero; point } =
         Pcs_batch.combine_split_commitments batch
+          ~reduce_with_degree_bound:(fun _ -> assert false)
```

Comment:
Can you comment the reason we use `assert false` please? In the case we reach this case once.

Reply:
Unreachable code